lib/command_runner.dart

```diff
@@ -320,7 +320,7 @@ abstract class Command<T> {
   /// The parsed argument results for this command.
   ///
   /// This will be `null` until just before [Command.run] is called.
```

Comment:
Probably worth updating the docs here (if we keep this change). `'This will throw if called before...'`.